### PR TITLE
 [4.1.x] Fix listeneronreconnecttest test   API-1196

### PR DESCRIPTION
**Pull request by srknzl**
The same description at https://github.com/hazelcast/hazelcast-nodejs-client/pull/1203 applies to this pr as well

### CODE REVIEW

File: test/TestUtil.js [hunk 1]
```diff
@@ -125,6 +125,37 @@ exports.markEnterprise = function (_this) {
     }
 };
 
+/**
+ * Duplicated this from BuildInfo because the logic of it is changed in 5.1, and in backward compatibility tests older
```
Comment (mdumandag) [inline test/TestUtil.js:129]: Did we correct this in the actual code for 4.1.x? If so, we don't need this

File: test/integration/ListenersOnReconnectTest.js [hunk 2]
```diff
@@ -88,69 +107,85 @@ describe('ListenersOnReconnectTest', function () {
...
+            const deferred = deferredPromise();
+            const member = await RC.startMember(cluster.id);
+            client = await Client.newHazelcastClient({
+                clusterName: cluster.id,
+                network: {
+                    smartRouting: isSmart
+                },
+                properties: {
+                    'hazelcast.client.heartbeat.interval': 1000
+                }
```
Comment (mdumandag) [inline test/integration/ListenersOnReconnectTest.js:144]: leftover?

File: test/TestUtil.js (hunk 1 above)
Comment (srknzl) [reply test/TestUtil.js:129]: We don't need this method anyway. I thought I deleted it. Let me delete it.

File: test/integration/ListenersOnReconnectTest.js (hunk 2 above)
Comment (srknzl) [reply test/integration/ListenersOnReconnectTest.js:144]: yep removing

**Review by mdumandag — APPROVED** [review]

**Review by yuce — APPROVED** [review]

**Pull request merged**